src/components/HeatmapCell.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const TableCell = styled.td`
+  text-align: center;
+  width: 39px;
+  height: 39px;
+  padding: 0;
+  color: white;
+  font-weight: bold;
+  font-size: ${({ theme }) => theme.font.size.small};
+  background-color: ${({ theme, colorValue }) => theme.color.count[colorValue]};
+  border: 1px solid ${({ theme, colorValue, clicked }) => (
+    clicked ? theme.color.tableRowHeader : theme.color.count[colorValue]
+  )};
+  &:hover {
+    border: 1px solid ${({ theme }) => theme.color.tableRowHeader};
+  }
+`;
+
+const HeatmapCell = ({
```

Comment:
Just a small suggestion :smile: You could make this component shorter by removing the `const`.

```
const HeatmapCell = ({
children, colorValue, handleCellClick, id, clickedCellId,
}) => (
    <TableCell
      colorValue={colorValue}
      clicked={clickedCellId === id}
      onClick={() => handleCellClick(id)}
    >
      {children}
    </TableCell>
  );
```

Reply:
I love these kinds of suggestions.  I think I get caught up in the tiny details sometimes and I forget to step back and see where I can make things more efficient.